### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by yoharden** [vercel/community#3781] [Account Issue] [answered]
### Summary

hello,my github account can't sign up.

### Steps to Reproduce

![%T BBV{UQ 795SDDS%L(U30](https://github.com/vercel/community/assets/128440606/e4930ce2-75a1-4320-aa37-7e4c9fcee06f)

**Comment by Code-Craft-Way** [comment]
I have the same.

**Comment by amyegan** [comment, marked as answer]
Hey there. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
